src/analysis/retail/hunter/marksmanship/modules/talents/LockAndLoad.tsx

```diff
@@ -77,15 +77,7 @@ class LockAndLoad extends Analyzer {
     this.totalProcs += 1;
     this.hasLnLBuff = true;
     if (this.spellUsable.isOnCooldown(TALENTS_HUNTER.AIMED_SHOT_TALENT.id)) {
-      const expectedCooldownDuration = this.abilities.getExpectedCooldownDuration(
-        TALENTS_HUNTER.AIMED_SHOT_TALENT.id,
-      );
-      if (expectedCooldownDuration) {
-        const newChargeCDR =
-          expectedCooldownDuration -
-          this.spellUsable.cooldownRemaining(TALENTS_HUNTER.AIMED_SHOT_TALENT.id);
-        this.spellUsable.reduceCooldown(TALENTS_HUNTER.AIMED_SHOT_TALENT.id, newChargeCDR);
-      }
+      this.spellUsable.endCooldown(TALENTS_HUNTER.AIMED_SHOT_TALENT.id, event.timestamp);
```

Comment:
Why is this being removed? Did functionality of LnL change?

Reply:
Unless I'm misunderstanding, isnt the logic here just removing/resetting the cooldown? 

Current logic - 
1. Get expected cooldown for aimed shot for current timestamp
2. If valid value, get the remaining cd
3. Reduce cooldown of aimed shot by the remaining cd ie resetting it

The outcome we want is to reset the cooldown of aimed shot so instead of doing the math to reduce it, i figured we just reset it and skip the math.

lmk if i missed anything tho

Reply:
I think this simplification is correct, but make sure to run some relevant logs through your local to sanity check that nothing changed in the module.